examples/exec-healthz/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM busybox
+MAINTAINER Prashanth B <beeps@google.com>
+ADD healthz_server healthz_server
+ADD README.md README.md
+EXPOSE 8080
```

Comment:
I would not EXPOSE here, and make the port a parameter - since we have no idea what context this will be used, it makes sense to stay flexible.  That said, I would accept a "TODO when needed".